node/test/util/merge_util.js

```diff
@@ -385,7 +385,7 @@ x=S:C2-1 s=Sa:a;C3-1 s=Sa:b;Bmaster=2;Bfoo=3`,
 Conflicting entries for submodule ${colors.red("s")}
 `,
             },
-            "conflict in submodule": {
+            "xxx conflict in submodule": {
```

Comment:
what's going on here?

Reply:
ahh, should have removed it. I used to add "xxx" in tests, so I can grep them more easily